tests/inference/test_fitting.py

```
@@ -653,3 +661,451 @@ def nll(params):
+
+# ============================================================================
+# _ProgressUpdater class tests
+# ============================================================================
+
+
+class TestProgressUpdater:
+    """Tests for _ProgressUpdater class."""
+
+    def test_update_sets_progress_and_postfix(self, mock_pbar):
+        """update() should set pbar.n and postfix with NLL."""
+        from everwillow.inference.fitting import _ProgressUpdater  # noqa: PLC2701
```

Ruff (0.14.13)

Unused `noqa` directive (non-enabled: `PLC2701`)

Remove unused `noqa` directive

(RUF100)